lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -52,6 +53,9 @@
       description: controls verification and validation of SSL certificates, mostly you only want to turn off with self signed ones.
       type: boolean
       default: True
+    namespace:
```

Comment:
Should not an `AnsibleError` be raised either when hvac doesn't handle namespace or when HashiCorp Vault service (<0.11) doesn't ?

Reply:
yeah, i am planning to add another try / catch to account for the older version tests (where namespaces isnt available)